packages/connect-web/README.md

```diff
@@ -55,3 +55,37 @@ for a tutorial, or take a look at [our example](https://github.com/connectrpc/co
 Connect plays nice with Vue, Svelte, Remix, Next.js, Angular and many others. Take a look at
 [our examples](https://github.com/connectrpc/examples-es) for various frameworks.
 
+# Tests
```

Comment:
I actually don't want to put this here, but just did for a discussion. Should we:
1. Omit this test section altogether from READMEs
2. Move some/all of this to `CONTRIBUTING`
3. Keep it here and just slim it down to some bare essentials
4. Keep it here as shown

I'm probably in favor of Step 1

Reply:
We should definitely not have this section in this README because it shows up on npmjs.com.

The make targets `testwebnode`, `testwebbrowser`, and `testwebbrowserlocal` each only run a single file `fetch.spec.ts`. This was useful when we ran the crosstest suite.

It should suffice to run this spec only on Node.js. That means we can delete:
- scripts "karma:serve", "karma"
- file karma.serve.conf.cjs
- targets `testwebbrowser`, `testwebbrowserlocal`

The only remaining related make target is `testwebnode`. Let's rename it to `testconnectwebpackage` for consistency with `testconnectnodepackage`.

The "Tests" section in the README can then be removed.

Reply:
I think the `karma.conf.cjs` file is also unused now so I deleted that too. So now only `karma.browserstack.conf.cjs` remains.